src/views/DocumentationTopic.vue

```diff
@@ -150,6 +151,7 @@ export default {
       showQuickNavigationModal: false,
       store: DocumentationTopicStore,
       BreakpointName,
+      enableMinimized: false,
```

Comment:
Since this is never actually expected to change during the lifecycle of an instance of this component, I would recommend just making it a prop with the `false` default—just like you already have it in the other component.